### ISSUE TITLE
ghostkit.io docs website - Infinite redirect from www to non-www

### ISSUE THREAD

**Issue by davidwebca** [nk-crew/ghostkit#209]
Hi!

I know I've mentioned this in a private support ticket, but since it's been a while, I just wanted to make sure it doesn't fall into a crack. The ghostkit.io docs website has not been accessible for months now, there's an infinite redirect from www.ghostkit.io to ghostkit.io (without www) and back again so it's impossible to view the docs. Gotta go on web archive to find them lately and that's not the best experience.

Keep up the great work 👍

**Comment by nk-o** [comment]
Hi David,

I remember your ticket, but I don't see this bug. I remember that I tried to test the site, and it is working perfectly for me.

We have redirections from old URLs to new ones. For example, this is a check result from one of the tools:

![Shot 2024-03-21 at 11 53 45](https://github.com/nk-crew/ghostkit/assets/14810891/e3aafeb3-98cd-4c87-abaa-49e34a820019)

It looks fine. As you can see, there is a redirect from non-www to www and from the old slug to the new one.

Regards,
Nikita.

**Comment by davidwebca** [comment]
Hi! As I saw your notification just a few minutes ago, I went to the website in Safari and it worked... until it didn't. I think I hit "https://ghostkit.io" and then it started again, but I can't reproduce this on Chrome at the moment, even though I could before and the redirects were happening on Chrome too... now it only happens in Safari 🤔 I'm very confused. But when I see it, it's clearly bouncing back and forth between the www and non-www of the domain. Even when I flush the browser's cache and my computer's DNS etc.

https://github.com/nk-crew/ghostkit/assets/412352/f03962f7-7bf2-4f90-8a43-864dcc50a403

There might be some issues with Cloudflare, but that's extremely strange. My friend and I have this problem (both in Canada, could be a regionalized bug with Cloudlfare) and we inconsistently reproduce it on our different devices. It doesn't happen on Safari on my iPad Pro connected to the same network, but it does happen on my iPhone. I'm stumped!
